_inc/lib/admin-pages/class.jetpack-admin-page.php

```diff
@@ -328,9 +327,31 @@ public static function wrap_ui( $callback, $args = array() ) {
 			echo $callback_ui;
 			?>
 			<!-- END OF CALLBACK -->
-			<div class="jp-footer">
+			<?php self::render_footer(); ?>
+		</div>
+		<?php
+		return;
+	}
+
+	/**
+	 * Render the footer of the jetpack dashboard. For admin pages.
+	 */
+	function render_footer() {
+		$admin_url = admin_url( 'admin.php?page=jetpack' );
+
+		$is_connected = Jetpack::is_active() || Jetpack::is_development_mode();
```

Comment:
`$is_connected` may end up being a bit confusing here, since it's not just about connected sites. Would `$is_dev_mode_or_connected` be too long?